### ISSUE TITLE
question about download_data.sh

### ISSUE THREAD

**Issue by LZDSJTU** [charlesq34/pointnet#105]
Hello, I want to ask if the dataset downloaded by download_data.sh is S3DIS or other dataset.

**Comment by bingo830422** [comment]
download_data.sh is for downloading HDF5 files for indoor 3d semantic segmentation.
If you want to download S3DIS data, you can download from http://buildingparser.stanford.edu/dataset.html.
This description is in README.md in sem_seg folder.

**Comment by LZDSJTU** [comment]
So I want to know what the indoor 3d semantic segmentation data downloaded by download_data.sh is. What is the name of the dataset?

**Comment by bingo830422** [comment]
indoor3d_sem_seg_hdf5_data

**Comment by LZDSJTU** [comment]
However, I do not find this dataset in other places. Is it a public dataset?

**Comment by abhishek-v** [comment]
What @LZDSJTU means is that the S3DIS dataset has been stored as HDF5 files

**Comment by LZDSJTU** [comment]
yeah! @abhishek-v So is indoor3d_sem_seg_hdf5_data the preprocessing result of S3DIS?

**Comment by bingo830422** [comment]
Yes!
If you download the raw data of S3DIS(which contains .txt files), you can also produce “indoor3d_sem_seg_hdf5_data”(which contains several .h5 files) by running the “collect_indoor3d_data.py” and “gen_indoor3d_h5.py”.
The hdf5 data is the feeding of the training. However, after training, if you want to visualize the testing result by running the “batch _inferece.py”, you need the .npy files. The numpy array files can be produced by running the “collect_indoor3d_data.py”

Hope this will answer your question.

**Comment by anniepapalexiou** [comment]
> download_data.sh is for downloading HDF5 files for indoor 3d semantic segmentation.
> If you want to download S3DIS data, you can download from http://buildingparser.stanford.edu/dataset.html.
> This description is in README.md in sem_seg folder.

This link is not valid anymore. At least it doesn't work for me. Do you have any idea where I can find those data please?

**Comment by NickSadjoli** [comment]
Hi, as mentioned previously, the [original link](http://buildingparser.stanford.edu/dataset.html) to the raw S3DIS dataset seem to be dead. This has caused issues for other projects that rely on the raw dataset, as seen in one of the issue pages for the Pytorch implementation of Pointnet & Pointnet++ mentioned above.

It seems that the raw dataset has instead been moved to a Google Drive link, which is officially retrievable by filling this [Google Form](https://docs.google.com/forms/d/e/1FAIpQLScDimvNMCGhy_rmBA2gHfDu3naktRm6A8BPwAWWDv-Uhm6Shw/viewform?c=0&w=1). For those  wondering, I accidentally found this link from one of the [documentation page](https://torch-points3d.readthedocs.io/en/latest/_modules/torch_points3d/datasets/segmentation/s3dis.html) for Torch Points 3D, after some searching for the s3dis data. I'm not sure how they have the direct access to this S3DIS data, but that's my findings.

Hope this helps with everyone who'd like to use the raw S3DIS data and their projects!

**Comment by anniepapalexiou** [comment]
> Hi, as mentioned previously, the [original link](http://buildingparser.stanford.edu/dataset.html) to the raw S3DIS dataset seem to be dead. This has caused issues for other projects that rely on the raw dataset, as seen in one of the issue pages for the Pytorch implementation of Pointnet & Pointnet++ mentioned above.
> 
> It seems that the raw dataset has instead been moved to a Google Drive link, which is officially retrievable by filling this [Google Form](https://docs.google.com/forms/d/e/1FAIpQLScDimvNMCGhy_rmBA2gHfDu3naktRm6A8BPwAWWDv-Uhm6Shw/viewform?c=0&w=1). For those wondering, I accidentally found this link from one of the [documentation page](https://torch-points3d.readthedocs.io/en/latest/_modules/torch_points3d/datasets/segmentation/s3dis.html) for [Torch Points 3D](https://github.com/nicolas-chaulet/torch-points3d), after some searching for the s3dis data. I'm not sure how they managed to get this url, but that's my findings.
> 
> Hope this helps with everyone who'd like to use the raw S3DIS data and their projects!
> 
> EDIT: Added Torch Points 3D Github page

Thank you for your reply! Indeed this is the only link available now for downloading the raw S3DIS data. I also found this link after a lot of search and used the data, so I can verify that the data are indeed the correct ones.

**Comment by youssef962** [comment]
the shell file is not working with me and it says that it failed because of certificate issue! what should I do?!
